.github/workflows/propagate_to_manifestdb.yml

```diff
@@ -0,0 +1,77 @@
+name: Propagate update to OSBuild
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  propagate:
+    runs-on: ubuntu-latest
+    env:
+      SCHUTZBOT_GH: ${{ secrets.SCHUTZBOT_GH}}
+    steps:
+      - name: Checkout OSBuild
+        uses: actions/checkout@v3
+        with:
+          path: osbuild
+          fetch-depth: 0
+
+      - name: Update Manifest-db
+        run: |
+          # update only if manifest-db SHA was updated in the last commit
+          cd osbuild
+          ACTUAL_MDB_SHA=$(git show HEAD:Schutzfile | jq -r '.global.dependencies."manifest-db".commit')
+          PREVIOUS_MDB_SHA=$(git show HEAD^:Schutzfile | jq -r '.global.dependencies."manifest-db".commit')
+          if [[ $ACTUAL_MDB_SHA != $PREVIOUS_MDB_SHA ]] ; then
+
+            echo "updating the dependency in manifest-db"
+            OSBUILD_HEAD="${{ github.sha }}"
+            echo "Latest OSBuild SHA: $OSBUILD_HEAD"
+
+            # login as schutzbot
+            echo "${SCHUTZBOT_GH}" | gh auth login --with-token
+
+            cd ..
+            # Clone manifest-db
+            git clone https://github.com/osbuild/manifest-db.git
+
+
+            # Load the previous osbuild REF from manifest-db's Schutzfile
+            cd manifest-db
+            OLD_REF=$(jq -r '.global.dependencies.osbuild.commit' Schutzfile)
+
+            # OSBuild directory Generate the commit list between the two osbuild
+            # revisions for the PR
+            cd ../osbuild
```

Comment:
Moving this before the previous `cd ..` would save us a couple of `cd`s.

Reply:
We can't really save much `cd`s here, because, we need first to be in the `osbuild` repo to knwo is we should upgrade, then to hop in `manifest-db` to get the  `OLD_REF` commit from its `Schutzfile` and to finally be in the `osbuild` repo to generate the commit list.